File: Beecrowd/1 - Beginner/1101 - 1190/1101 - Sequence of Numbers and Sum/1101_SequenceofNumbersandSum.c
#include <stdio.h>

int main() {
    int number1, number2;

    while (scanf("%d %d", &number1, &number2) == 2 && number1 > 0 && number2 > 0) {
        int sum = 0;

        if (number1 > number2) {
            int temp = number1;
            number1 = number2;
            number2 = temp;
        }

        for (int i = number1; i <= number2; i++) {
            printf("%d ", i);
            sum += i;
        }

        printf("Sum=%d\n", sum);
    }

    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1101 - Sequence of Numbers and Sum/1101_SequenceofNumbersandSum.py
while True:
    numbers = input().split()
    number1 = int(numbers[0])
    number2 = int(numbers[1])

    if number1 > 0 and number2 > 0:
        sum_result = 0

        if number1 > number2:
            number1, number2 = number2, number1

        for i in range(number1, number2 + 1):
            print(i, end=" ")
            sum_result += i

        print(f"Sum={sum_result}")
    else:
        break
